frontend/src/Components/SavedGamesPagination/SavedGamesPagination.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import SavedNavigationButton from '../SavedNavigationButton/SavedNavigationButton';
+import SavedNavigationLink from '../SavedNavigationLink/SavedNavigationLink';
+
+const SavedGamesPagination = ({ totalPages, currentPage, firstPage, lastPage }) => {
+  return (
+    <nav aria-label='Saved games navigation'>
+      <ul className='pagination justify-content-center'>
+        <SavedNavigationButton navigation='«' navigationPage={currentPage-1} disabled={firstPage !== false} />
+        {Array(totalPages).fill(null).map((e, index) => (<SavedNavigationLink
+          key={`saveGame${index+1}`}

```

Comment:
I don't like how I am doing index+1 each time... should try to fix that with a variable if possible.